Add support for number of eeproms in the ethtool_modinfo structure.
This is needed to support more than one sfp eeproms in ganged ports.

diff --git a/include/linux/ethtool.h b/include/linux/ethtool.h
index 6d2ad9e..ff9d5ec 100644
--- a/include/linux/ethtool.h
+++ b/include/linux/ethtool.h
@@ -131,7 +131,8 @@ struct ethtool_modinfo {
 	__u32   cmd;
 	__u32   type;
 	__u32   eeprom_len;
-	__u32   reserved[8];
+	__u32   eeprom_nums;
+	__u32   reserved[7];
 };
 
 /**
